server/app/views/admin/programs/ProgramAdministratorProgramListView.java

```diff
@@ -116,32 +118,33 @@ public Tag renderProgramListItem(
                         StyleUtils.applyUtilityClass(StyleUtils.RESPONSIVE_MD, Styles.MR_4)))
             .withClasses(Styles.FLEX);
 
-    Tag midContent =
+    DivTag midContent =
         div(programDescriptionText)
             .withClasses(Styles.TEXT_GRAY_700, Styles.TEXT_BASE, Styles.MB_8, Styles.LINE_CLAMP_3);
 
-    Tag bottomContent =
+    DivTag bottomContent =
         div(
                 p(lastEditText).withClasses(Styles.TEXT_GRAY_700, Styles.ITALIC),
                 p().withClasses(Styles.FLEX_GROW),
                 maybeRenderViewApplicationsLink(viewApplicationsLinkText, activeProgram))
             .withClasses(Styles.FLEX, Styles.TEXT_SM, Styles.W_FULL);
 
-    Tag programDeepLink =
+    LabelTag programDeepLink =
         label("Deep link, use this URL to link to this program from outside of CiviForm:")
             .withClasses(Styles.W_FULL)
             .with(
                 input()
-                    .withValue(
+                    .attr(
```

Comment:
I believe this is incorrect. The new version of the code sets an attribute of the URL rather than setting the "value" attribute to the URL.

I think we should switch this back to `withValue`.